frontend/libs/client/feature-enrollment/src/lib/basic-data/basic-data.component.html

```diff
@@ -95,7 +107,7 @@ <h2>
             mat-button
             color="primary"
             type="button"
-            (click)="openContactDialog(date.toLocaleDateString('de'))"
+            (click)="openContactDialog(date.toLocaleDateString('de'), date)"
```

Comment:
Wenn das date benötigt wird, fände ich es schöner, hier nur das date zu übergeben und das .toLocaleDateString dann in der openContactDialog-Methode aufzurufen. Was meinst du?